app/controllers/miq_task_controller.rb

```diff
@@ -69,38 +66,22 @@ def jobs
                                                      :headers    => @view.headers})
         page << "miqSparkle(false);"  # Need to turn off sparkle in case original ajax element gets replaced
       end
-    else                      # Came in from non-ajax, just get the jobs
-      get_jobs(tasks_condition(@tasks_options[@tabform]))
     end
   end
 
-  def get_jobs(conditions)
+  def list_jobs
```

Comment:
something is nice about having the conditions explicitly passed into this method.

but it is duplicated in many places, so this will do